test/packettest.c

```diff
@@ -12,13 +12,13 @@
 
 #define BUF_LEN 255
 
-static unsigned char smbuf[BUF_LEN];
+static unsigned char smbuf[BUF_LEN + 1];
```

Comment:
It might have been nice to say why this was done with a /* */

Reply:
@slontis: Agreed. Will do in some next commit